app/src/main/java/com/android/voyageur/App.kt

```diff
@@ -1,28 +1,33 @@
 package com.android.voyageur
 
 import androidx.compose.runtime.Composable
+import androidx.lifecycle.viewmodel.compose.viewModel
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.navigation
 import androidx.navigation.compose.rememberNavController
+import com.android.voyageur.model.trip.TripsViewModel
 import com.android.voyageur.ui.navigation.NavigationActions
 import com.android.voyageur.ui.navigation.Route
 import com.android.voyageur.ui.navigation.Screen
-import com.android.voyageur.ui.overview.OverviewScreen
+import com.android.voyageur.ui.overview.AddTripScreen
 import com.android.voyageur.ui.profile.ProfileScreen
 import com.android.voyageur.ui.search.SearchScreen
 
 @Composable
 fun VoyageurApp() {
   val navController = rememberNavController()
   val navigationActions = NavigationActions(navController)
+  val tripsViewModel: TripsViewModel = viewModel(factory = TripsViewModel.Factory)
 
   NavHost(navController = navController, startDestination = Route.OVERVIEW) {
     navigation(
         startDestination = Screen.OVERVIEW,
         route = Route.OVERVIEW,
     ) {
-      composable(Screen.OVERVIEW) { OverviewScreen(navigationActions) }
```

Comment:
Please revert this. The first screen shown after login should be Overview Screen, not Add Trip Screen.